python/Ganga/GPIDev/Lib/GangaList/GangaList.py

```diff
@@ -156,6 +160,15 @@ def _on_attribute__set__(self, obj_type, attrib_name):
             self._is_a_ref = False
         return self
 
+    def _getParent(self):
```

Comment:
I'm planning to investigate in future if I can let subjobs have a GangaList as a parent which would allow for something like GangaLists of Jobs to be stored in Box repo (currently not possible).

In this case I will likely be wanting to tie into the getParent/setParent functions at least during development. It's not required to keep it here, but I'm playing around with this interface and it saves me the time of adding it back in, in the future just to test that it's working with a debug statement.